src/coreclr/jit/llvmcodegen.cpp

```diff
@@ -1118,6 +1118,7 @@ void Llvm::visitNode(GenTree* node)
             break;
         case GT_JMP:
             NYI("LLVM/GT_JMP"); // Requires support for explicit tailcalls.
+            LLVM_FALLTHROUGH;
```

Comment:
```suggestion
            break;
```
(Let's see if this angers MSVC's code analyzer... It can be `FALLTHROUGH;` if it does.)

We don't use LLVM macros in Jit source code.